test/helpers/kubectl.go

```diff
@@ -252,7 +253,8 @@ func (kub *Kubectl) WaitforPods(namespace string, filter string, timeout time.Du
 func (kub *Kubectl) WaitForServiceEndpoints(namespace string, filter string, service string, port string, timeout time.Duration) (bool, error) {
 	body := func() bool {
 		var jsonPath = fmt.Sprintf("{.items[?(@.metadata.name =='%s')].subsets[0].ports[0].port}", service)
-		data, err := kub.GetEndpoints(namespace, filter).Filter(jsonPath)
+		cmd := kub.GetEndpoints(namespace, filter)
```

Comment:
Can you rename this from `cmd` to `res`? It adheres to the convention set throughout the Ginkgo framework.